framework/include/dm/dm_connectivity.h

```diff
@@ -35,7 +35,11 @@
 extern "C"
 {
 #endif
-
+/**
+ * @brief Specify the data structure to hold WiFi scanning information.
+ * The information comprises SSID and BSSID of nearby Access Points (APs),
```

Comment:
In brief, you should add simple sentence. If you have more description, you should use details.

Reply:
for example)
@brief Specify the data ~
@details The information~~